packages/contracts/deploy/05_ens.ts

```diff
@@ -0,0 +1,36 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+import {ENS_ADDRESSES, getContractAddress, setupENS} from './helpers';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts, ethers, getChainId, network} = hre;
+  const {deploy} = deployments;
+
+  const {deployer} = await getNamedAccounts();
+
+  const managingDAOAddress = await getContractAddress('DAO', hre);
+  const node = ethers.utils.namehash('dao.eth');
```

Comment:
I think we can reuse this from an `util` source file that should be merged or about to be